# Task breakdown for robots
uint8 PART=0
uint8 TRAY=1


uint8 task_type # PART OR TRAY
int8 tray_id
ariac_msgs/Part part
geometry_msgs/Pose pose
string order_id
uint8 quadrant
uint8 agv_number



